src/DockerFileGenerator.php

```diff
@@ -92,41 +92,12 @@ public function __construct(
 
     public function generateStub(): string
     {
-        $contents = self::FILE_TEMPLATE;
-
-        $contents = str_replace(
-            '__BASE_PHP_IMAGE_TOKEN__',
-            $this->image,
-            $contents,
-        );
-
-        $contents = str_replace(
-            '__PHP_EXTENSIONS_TOKEN__',
-            [] === $this->extensions
-                ? '[]'
-                : sprintf(
-                    '["%s"]',
-                    implode(
-                        '", "',
-                        $this->extensions,
-                    ),
-                ),
-            $contents,
-        );
-
-        $contents = str_replace(
-            '__PHAR_FILE_PATH_TOKEN__',
-            $this->sourcePhar,
-            $contents,
-        );
-
-        $contents = str_replace(
-            '__PHAR_FILE_NAME_TOKEN__',
-            basename($this->sourcePhar),
-            $contents,
-        );
-
-        return $contents;
+        return strtr(self::FILE_TEMPLATE, [
```

Comment:
I would put a line return there:

```suggestion
        return strtr(
            self::FILE_TEMPLATE,
            [
                //....
            ],
        );
```

mixed arguments alignments like that make it a bit hard to read IMO

Reply:
I agree, the generic way to solve that is to set up an automated checker though. It can even review your PRs for you ;-)